source/lib/header.js

```diff
@@ -0,0 +1,5 @@
+export const metatags = [
+  { 'name': 'description', 'content': 'Make your voice, time, effort and money all count for the causes you love and see your impact on your Giving Footprint with everydayhero. Get started now!' }
+]
+
+export const links = [{ 'rel': 'icon', 'href': `${process.env.BASE_PATH}/favicon.ico` }]
```

Comment:
If this asset URL get's handled by webpack then you don't need to worry about BASE_PATH (I can recommend forgetting it's existence at every opportunity)